docs/modules/plugins/pages/plugin-rest-api.adoc

```diff
@@ -251,6 +345,33 @@ When I execute HTTP GET request for resource with URL `https://example.com`
 
 === Wait for JSON element in the HTTP response
 
+Waits for a specified amount of time until HTTP response body contains an element by the specified JSON path. The actions of the step:
+
+. Execute sub-steps
+. Check if HTTP response is present and response body contains an element by JSON path
+. If the required JSON element exists or the maximum number of retries is reached, then the execution stops, otherwise the step actions are repeated
+. Stop step execution if HTTP response is not present or JSON element is found, otherwise sleep for the calculated part of specified duration and repeat actions from the start
```

Comment:
```suggestion
. Execute sub-steps.
. Check if the HTTP response is present and the response body contains an element by JSON path.
. If the required JSON element exists or the maximum number of retries is reached, then the execution stops, otherwise the step actions are repeated.
. Stop step execution if HTTP response is not present or JSON element is found, otherwise sleep for the calculated part of specified duration and repeat actions from the start.
```